Makefile

```
@@ -1,11 +1,17 @@
-.PHONY: build test clean run
+.PHONY: build test clean run lint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)